Creating_a_TDx_Feed.md

```diff
@@ -0,0 +1,64 @@
+# Creating a TDx Feed
+This documents contains information to assist in creating a TDx data feed, such as the feed format, business rules, and validation tools.
+
+## Feed Format and File Type
+TDx feeds are formatted according to the [GeoJSON](https://geojson.org/) specification. The output of a TDx feed is a GeoJSON document (a `.geojson` file) that contains a single [GeoJSON FeatureCollection](https://datatracker.ietf.org/doc/html/rfc7946#section-3.3) which includes information about the feed (see the [FeedInfo Object](/spec-content/objects/FeedInfo.md)) and a list of [GeoJSON Feature](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2)s describing entities specific to the type of feed, such as work zones.
```

Comment:
Should this section reference some kind of restriction instead of work zones? (last sentence)